Can't build main.

```zig
zig build-exe src/main.zig -lc -lX11 -lXft -I/usr/include/freetype2
/usr/lib/zig/std/fmt.zig:568:17: error: cannot format array without a specifier (i.e. {s} or {any})
                @compileError("cannot format array without a specifier (i.e. {s} or {any})");
                ^
/usr/lib/zig/std/fmt.zig:352:23: note: called from here
        try formatType(
                      ^
/usr/lib/zig/std/io/writer.zig:34:34: note: called from here
            return std.fmt.format(self, format, args);
                                 ^
/usr/lib/zig/std/debug.zig:68:27: note: called from here
    nosuspend stderr.print(fmt, args) catch return;
                          ^
./src/main.zig:319:19: note: called from here
            print ("Title of client {} read as verbatim string\n", .{cl.name});
                  ^
./src/main.zig:291:96: note: called from here
fn manageWindow(window: c.Window, window_attrs: c.XWindowAttributes, display: *c.Display) void {
                                                                                               ^
/usr/lib/zig/std/fmt.zig:568:17: error: cannot format array without a specifier (i.e. {s} or {any})
                @compileError("cannot format array without a specifier (i.e. {s} or {any})");
                ^
/usr/lib/zig/std/fmt.zig:352:23: note: called from here
        try formatType(
                      ^
/usr/lib/zig/std/io/writer.zig:34:34: note: called from here
            return std.fmt.format(self, format, args);
                                 ^
/usr/lib/zig/std/debug.zig:68:27: note: called from here
    nosuspend stderr.print(fmt, args) catch return;
                          ^
./src/main.zig:11:10: note: called from here
    print("Message arrived: {}\n ", .{message});
         ^
./src/main.zig:10:70: note: called from here
fn process_client_message(message: [20]u8, display: *c.Display) void {
                                                                     ^
make: *** [makefile:2: main] Error 1
```

--- a/src/main.zig
+++ b/src/main.zig
@@ -8,14 +8,14 @@ const print = std.debug.warn;
 const allocator = std.heap.c_allocator;
 
 fn process_client_message(message: [20]u8, display: *c.Display) void {
-    print("Message arrived: {}\n ", .{message});
+    print("Message arrived: {s}\n ", .{message});
     switch(message[0]){
         'k' =>
             _ = if (focus) |f| { _ = c.XKillClient(display, f.win);
             print("Killing client {}\n", .{f.win});
         },
         else => {
-            print("Unknown message recived: {}\n", .{message});
+            print("Unknown message recived: {s}\n", .{message});
         },
     }
 }
@@ -316,9 +316,9 @@ fn manageWindow(window: c.Window, window_attrs: c.XWindowAttributes, display: *c
         const XA_STRING : c.Atom = 31; 
         if (tp.encoding == XA_STRING){
             std.mem.copy(u8, cl.name[0..25], tp.value[0..25]); 
-            print ("Title of client {} read as verbatim string\n", .{cl.name});
+            print ("Title of client {s} read as verbatim string\n", .{cl.name});
         } 
-        print("Title of client is now {}\n", .{ cl.name});
+        print("Title of client is now {s}\n", .{ cl.name});
     }
     _ =  c.XSelectInput(display, window, 0
         | c.FocusChangeMask  // FocusIn, FocusOut */

